spec/identity/index.html

```diff
@@ -476,18 +476,18 @@ <h2 id="h2_terminology"><span class="secno">2. </span>Terminology</h2>
 <dd>The Requesting Agent initiates a request to a Service listening on a specific port using a given protocol on a given Server.</dd>
 
 <dt><dfn title="Server" id="dfn-server">Server</dfn></dt>
-<dd>A Server is a machine contactable at a domain name or IP address that hosts a number of globally accessible Services.</dd>
+<dd>A Server is a machine, contactable at a domain name or IP address, that hosts Services which are accessible over the network.</dd>
```

Comment:
To allow for cloud as well as on-site deployments...
```suggestion
<dd>A Server is a physical or virtual machine, contactable at a domain name or IP address, that hosts Services which are accessible over the network.</dd>
```